ruby/game/hangman.rb

```diff
@@ -0,0 +1,83 @@
+# make a class
+class Game
+# add attributes for target_word, guessed_letters, guessed_word, number_of_guesses
+	attr_reader :number_of_guesses
+	attr_accessor :target_word, :guessed_letters, :guessed_word
+
+# add methods for getting the target_word, guessing letters
+	# initalize setting the guessed_letters and guessed_word to empty arrays
+	def initialize
```

Comment:
this is possibly a design decision but i feel like we should initialize a game with the "secret_word" or "target_word"